docs/static/releasenotes.asciidoc

```diff
@@ -3,6 +3,7 @@
 
 This section summarizes the changes in the following releases:
 
+* <<Logstash-6-4-0,Logstash 6.4.0>>
```

Comment:
`<<Logstash` should be `logstash` (lowercase).  The link's not resolving and doc won't build.

Reply:
@karenzone oops - I've pushed a commit that fixes this